ee/modules/110-istio/monitoring/prometheus-rules/revisions.yaml

```diff
@@ -69,8 +69,34 @@
         ### pod-wide configuration
         kubectl -n {{$labels.namespace}} get pods -l istio.io/rev={{$labels.desired_revision}}
         ```
-  - alert: D8IstioDataPlanePatchVersionMismatch
-    expr: max by (namespace, revision, actual_sidecar_image_tag) (d8_istio_data_plane_patch_version_mismatch == 1)
+  - alert: D8IstioDataPlaneVersionMismatch
+    expr: |
+      (
```

Comment:
For complicated promql, I'd like to see an exaplanation about its parts. If a new developer needs to add something to this query or alter it, it will take some time to figure out what is happening.

Reply:
Done.